runtime/drivers/duckdb/migrate.go

```diff
@@ -20,27 +22,33 @@ var migrationVersionTable = "rill.migration_version"
 // Migrate implements drivers.Connection.
 // Migrate for DuckDB may not be safe for concurrent use.
 func (c *connection) Migrate(ctx context.Context) (err error) {
-	// Create rill schema if it doens't exist
-	_, err = c.db.ExecContext(ctx, "create schema if not exists rill")
+	conn, release, err := c.acquireMetaConn(ctx)
+	if err != nil {
+		return err
+	}
+	defer func() { _ = release() }()
```

Comment:
why ignore the error from release ? Should we log it in case there is any connection leak ?

Reply:
I guess we should. Looking in `database/sql`, it looks like the only error it can return is if you try to close the conn twice, so not handling it shouldn't lead to any leaks currently

Reply:
Ok I was looking for a similar thing but didn't knew about it

Reply:
I guess Go code does not throw any runtime exceptions/errors like Java so it can be ignored and in case of exceptional situations like low memory etc. it should already panic and print in the logs? Only caveat is if in future if API changes and start throwing new error but not a priority to handle now, can be added in the backlog.

Reply:
Yeah, panics will always propagate. Generally, it's good to handle all errors and only ignore specific named errors, since as you mention, more errors may be returned in future. But for `defer` statements, I often see people pick simpler code. I added a backlog to add good handling for all defer errors.